package/MDAnalysis/lib/util.py

```diff
@@ -295,7 +295,7 @@ def anyopen(datasource, mode='r', reset=True):
                     stream.reset()
                 except (AttributeError, IOError):
                     try:
-                        stream.seek(0L)
```

Comment:
@richardjgowers I noticed that function (because `sys.maxint` does not exist in Python 3 where integers are as long as they need to be). I do not see why it would fail with a specific hardcoded value like `0`. In python 2, `seeksize` should always be a `long`, regardless of the type of `nframes`.